src/main/java/models/StockerKey.java

```diff
@@ -0,0 +1,101 @@
+package models;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.Column;
+
+/**
+ * La classe StockerKey représente une clé composite pour l'entité Stocker,
+ * composée de IdMagasin et IdService.
+ */
+public class StockerKey implements Serializable {
+
+    @Column(name = "IdMagasin")
+    private int IdMagasin;
```

Comment:
```suggestion
    private int idMagasin;
```

Penser à renommer dans tout le fichier